source/classes.tex

```diff
@@ -4837,10 +4906,10 @@
 left operand (considered as a pointer in the
```

Comment:
old line 4814 is missing a change of 'named' to 'designated'

Reply:
fixed